cmd/gitops/app/remove/cmd.go

```diff
@@ -48,7 +50,17 @@ func runCmd(cmd *cobra.Command, args []string) error {
 	params.Name = args[0]
 	params.Namespace, _ = cmd.Parent().Flags().GetString("namespace")
 
-	appService, appError := apputils.GetAppService(ctx, params.Name, params.Namespace)
+	kube, _, err := kube.NewKubeHTTPClient()
```

Comment:
I don't understand why we would want to introduce so much redundant code. It trigger my DRY detector. If any of those steps change, all sorts of commands need to be updated. Additionally, people writing commands have to know a lot more about the underlying support.

Reply:
@jrryjcksn Good call, updated to move that stuff into a helper. The main impetus for this change was to remove the two `GetAppService` functions that were **almost** identical. PTAL.